### PR TITLE
Add tag to button component

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
Closes : https://github.com/zakodium-oss/react-science/issues/649

### CODE REVIEW

File: src/components/button/Button.tsx [hunk 1]
```diff
@@ -27,9 +32,33 @@ export function Button(props: ButtonProps) {
...
+                position: 'absolute',
+                top: 0,
+                left: 0,
+                cursor: 'default',
+                fontSize: '0.875em',
+                padding: 2,
+                minHeight: 15,
+                minWidth: 15,
+                lineHeight: '1em',
+                flexDirection: 'column-reverse',
```
Comment (stropitek) [inline src/components/button/Button.tsx:48]: Is this really useful?

**Review by stropitek — COMMENTED** [review]
- Allow `tagProps` to be passed to toolbar component as well. If the `style` prop is passed to `tagProps`, it should probably not override things like "absolute" which are in the default `style`.
- In button and toolbar stories, do not show example with tag in control. Show tag in specific story.
- Stories with longer tags than 1 chars. Up to 3 chars for the toolbar story to see how it looks like in that case.

File: src/components/button/Button.tsx (hunk 1 above)
Comment (wadjih-bencheikh18) [reply src/components/button/Button.tsx:48]: Yes to make it centred

Comment (stropitek) [reply src/components/button/Button.tsx:48]: `align-text: center;` is better then (mainly because more explicit about why the property is there)

Comment (wadjih-bencheikh18) [reply src/components/button/Button.tsx]: I used `flexDirection: 'column'` because the default was `flexDirection: 'row'`, and nothing make it centered

Comment (stropitek) [reply src/components/button/Button.tsx]: It works but isn't it clearer to just use the `align-text` property in order to align (center) the text?

Comment (stropitek) [reply src/components/button/Button.tsx]: Using the dev tools it seems to work as well

**Review by stropitek — APPROVED** [review]

**Comment by stropitek** [comment]
@wadjih-bencheikh18 There is a conflict left to resolve

**Comment by wadjih-bencheikh18** [comment]
@stropitek conflict resolved

**Pull request merged**